skiko/src/jvmMain/kotlin/org/jetbrains/skiko/SkiaLayer.jvm.kt

```diff
@@ -134,9 +159,12 @@ actual open class SkiaLayer internal constructor(
     val windowHandle: Long
         get() = backedLayer.windowHandle
 
+    private var _isFullscreen = false
     actual var fullscreen: Boolean
-        get() = backedLayer.fullscreen
+        get() = _isFullscreen
```

Comment:
This way `backedLayer.fullscreen` becomes bogus, we may want to rename it to something like `fun requestPlatformFullscreen(newValue: Boolean)` and `fun checkPlatformFullscreen(): Boolean`.

Reply:
> something like fun requestPlatformFullscreen(newValue: Boolean) and fun checkPlatformFullscreen(): Boolean

I don't agree that we should introduce these functions on SkiaLayer level. We can discuss online.

The purpose of the property is to simplify the interaction with the native window, as many mainstream frameworks do.